submissions/solaryasha/html-css-popup/index.html

```diff
@@ -0,0 +1,224 @@
+                                    <figcaption>Contacts</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/tanslation.png" alt="translate">
+                                    <figcaption>Translate</figcaption>
+                                </figure>
+                            </a>
+                        </li>                       
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/videos.png" alt="videos">
+                                    <figcaption>Videos</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/search.png" alt="search">
+                                    <figcaption>Search</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/photos.png" alt="photo">
+                                    <figcaption>Photo</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/apps.png" alt="apps">
+                                    <figcaption>Apps</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                        
+                        <li class="popupItem">
+                            <a href="#">
+                                <figure>
+                                    <img src="img/documents.png" alt="documents">
+                                    <figcaption>Documents</figcaption>
+                                </figure>
+                            </a>
+                        </li>
+                    </ul>
+                    
+                    <input type="checkbox" id="buttonMore" class="moreApps">
+                    <label for="buttonMore" class="menuMoreButton"> More</label>
+                    
+                    <hr class="dividingLine">
```

Comment:
You can use border of block as divider without extra HTML element.
I mean use CSS for visual representing. Avoid adding extra elements only for visual representing.

Reply:
Thanks for the tip!